src/Microsoft.IdentityModel.Xml/Reference.cs

```diff
@@ -148,9 +148,9 @@ public void Verify(CryptoProviderFactory cryptoProviderFactory)
                     new MessageDetail(
                         LogMessages.IDX30201,
                         Uri ?? Id),
-                    ValidationFailureType.XmlValidationFailed,
                     typeof(XmlValidationException),
-                    new System.Diagnostics.StackFrame());
+                    new System.Diagnostics.StackFrame(),
```

Comment:
why do we use System.Diagnostics.StackFrame() here instead of ValidationError.GetCurrentStackFrame()?

Reply:
Because this code pre-dates the changes to use `GetCurrentStackFrame()`. In one of the draft PRs, for signature and algorithm extensibility, this is changed to use `GetCurrentStackFrame()`.